### DISCUSSION TITLE
Gradient on circular progress bar color

### DISCUSSION THREAD

**Discussion by Sirrom84** [chakra-ui/chakra-ui#6534] [Q&A]
Hi, 

I wondered if anybody knew how to add a linear gradient to the colour of the circular progress bar. I was able to add a gradient to the regular progress bar by using the sx attribute like so:

```
 sx={{
          "& > div": {
            background:
              "linear-gradient(89.95deg, #0A1B57 1.03%, #0A3296 26.79%, #1E64F0 51.52%, #41BBFB 75.74%, #88CCEE 99.96%)",
          },
        }}
```
but this is not working on the circular bar. Any help or suggestions would be appreciated

**Comment by Rezaabdi78** [comment]
As far as I know, when inspecting the component, the property `color` is passed to determine the progress color, is used as `stroke`.

So you can not use the same method you've provided for the `CircularProgress`. 

The most straightforward solution to this is implementing a custom component for that. 

I have the same problem and couldn't find any better solution yet. Let me know if you overcome this problem, please :)
